setuptools/_distutils/command/bdist_dumb.py

```diff
@@ -90,7 +90,7 @@ def run(self):
 
         # And make an archive relative to the root of the
         # pseudo-installation tree.
-        archive_basename = "%s.%s" % (self.distribution.get_fullname(),
+        archive_basename = "{}.{}".format(self.distribution.get_fullname(),
```

Comment:
It looks like the tool breaks hanging indents. It's probably acceptable for codebases that use something like black to autoformat, but this project hasn't been able to adopt black yet, so it'll require manual reformatting of affected files.